### PR TITLE
[MU4] Added build vst module on CI

### PR DESCRIPTION
**Pull request by igorkorsukov**
Added build vst module on CI

except for Windows

### CODE REVIEW

File: .github/workflows/ci_linux.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: CI_Linux
 on:
   pull_request:
     branches:
-    - master  
+    - master_off
```
Comment (vpereverzev) [inline .github/workflows/ci_linux.yml:6]: I'm curious what does it mean

File: build/ci/linux/build.sh [hunk 2]
```diff
@@ -48,11 +48,25 @@ ${CC} --version
...
+BUILD_VST=OFF
+else
+BUILD_VST=ON
+fi
 
 echo "=== BUILD === "
 
 make revision
-make -j2 MUSESCORE_BUILD_CONFIG=$MUSESCORE_BUILD_CONFIG BUILD_NUMBER=$BUILD_NUMBER TELEMETRY_TRACK_ID=$TELEMETRY_TRACK_ID BUILD_UI_MU4=${BUILD_UI_MU4} BUILD_UNIT_TESTS=ON portable
+make -j2 MUSESCORE_BUILD_CONFIG=$MUSESCORE_BUILD_CONFIG \
```
Comment (vpereverzev) [inline build/ci/linux/build.sh:62]: hm, it's interesting why it's -j2
BTW how many processing threads do we have on GitHub workers?

Comment (igorkorsukov) [reply build/ci/linux/build.sh:62]: 2

File: .github/workflows/ci_linux.yml (hunk 1 above)
Comment (igorkorsukov) [reply .github/workflows/ci_linux.yml:6]: I switched off, while testing (WIP)

File: build/ci/linux/build.sh (hunk 2 above)
Comment (Jojo-Schmitz) [reply build/ci/linux/build.sh:62]: May still increase speed to use `-j 4` or just `-j`

**Pull request merged**